pom.xml

```diff
@@ -2003,7 +2003,7 @@
                             <phase>validate</phase>
                             <configuration>
                                 <configLocation>
-                                    https://raw.githubusercontent.com/wso2/code-quality-tools/v1.1/checkstyle/checkstyle.xml
+                                    https://raw.githubusercontent.com/wso2/code-quality-tools/v1.2/checkstyle/checkstyle.xml
```

Comment:
with this we can remove encoding="UTF-8" also. right?

Reply:
yes, +1.